### PR TITLE
Widget sidebar 2

### PR DESCRIPTION
**Pull request by Dave-Gallup**
Adds sidebars to individual widgets and widget delete functionality.

### CODE REVIEW

**Comment by Dave-Gallup** [comment]
I altered `index.html` and added some inline css styling which I know we need to handle better.  I'll need to speak with you about alternatives.

File: .env.github-api [hunk 1]
```diff
@@ -0,0 +1,8 @@
+GITHUB_CLIENT_ID=aac2c89a1b7f5749c3db
```
Comment (HamidAghdaee) [inline .env.github-api]: @joshwyatt I'm not sure why this change shows up here., and a similar change shows up in @SSabu's PR. These have already been merged as part of the mob code session as far as I know. Do you know what happened or what to do with it?

File: .gitignore [hunk 2]
```diff
@@ -15,9 +15,9 @@
 .env.development.local
 .env.test.local
 .env.production.local
-.env.slack-api
-.env.transit-api
-.env.github-api
+.env.slack
```
Comment (HamidAghdaee) [inline .gitignore]: @SSabu Please make sure the .gitgnore file you have here is exactly the same as upstream/dev. This would also fix the issue of the addition of the .env files I see at the top here

Comment (HamidAghdaee) [reply .gitignore]: I meant @Dave-Gallup. But this goes for @SSabu as well
